src/System.Private.ServiceModel/tests/Benchmarks/WCFCorePerf/benchmarks.wcfcore.Perf.yml

```diff
@@ -1,83 +1,60 @@
 jobs:
-  BasicHttpWithBuffered:
+  WCFCorePerf:
     source:
       repository: https://github.com/dotnet/wcf.git
       branchOrCommit: master
       project: src/System.Private.ServiceModel/tests/Benchmarks/WCFCorePerf/WCFCorePerf/WCFCorePerf.csproj
+    readyStateText: WCFCorePerf Client
```

Comment:
I didn't realize this was a perf client and not a service, due to the name of the job. The `reasyStateText` might not be necessary them, but you can keep it as it shouldn't hurt.

However I saw that in this app you are generating some measurements. The names are odd: `channelopen`, `firstrequest` , `bombardier/requests`, `bombardier/rps/max`. And ideally anything that is coming from your own client should be prefixed with a descriptor of this client.

For instance the "bombardier" ones, these names come from a specific HTTP load client named "bombardier", this is a prefix for its metrics, there is no reason to use that. Since your client is called `WCFCorePerf` you could use a name like `wcfcoreperf/rps/mean` instead.

Reply:
I have removed reasyStateText and changed the name prefix for measurements in commit [cb04d79](https://github.com/dotnet/wcf/pull/4394/commits/cb04d796e5cf53e0f94211fb2370382039678ffc)